crates/web/frontend/src/components/shortcut-popup/shortcut-popup.tsx

```diff
@@ -0,0 +1,68 @@
+import { Keyboard, X } from "lucide-react";
+import ActionButton from "../action-button/action-button";
+import {
+	Dialog,
+	DialogContent,
+	DialogDescription,
+	DialogHeader,
+	DialogTitle,
+	DialogTrigger,
+} from "../ui/dialog";
+import { useState } from "react";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "../ui/table";
+
+const ShortcutPopup = () => {
+	const [open, setOpen] = useState(false);
+
+	return (
+		<Dialog open={open} onOpenChange={setOpen}>
+			<DialogTrigger asChild>
+				<ActionButton description="Show keyboard shortcuts" className="p-3">
+					<Keyboard className="w-4 h-4" />
+				</ActionButton>
+			</DialogTrigger>
+			<DialogContent className="w-[34rem] max-w-[80vw]">
+				<X
+					className="absolute top-4 right-4 h-4 w-4 cursor-pointer"
+					onClick={() => setOpen(false)}
+				/>
+				<DialogHeader>
+					<DialogTitle>Playground shortcuts</DialogTitle>
+					<DialogDescription>
+						Check all available keyboard shortcuts to improve your efficiency
+					</DialogDescription>
+				</DialogHeader>
+				<Table>
+					<TableHeader>
```

Comment:
Can you create something like in the examples?

Have the rows in a js array and then to a `array.map` to define the rows. So you dont have to repeat yourself so much